index.html

```diff
@@ -158,9 +181,22 @@
             width: 1.5rem;
         }
 
-        path, rect, circle {
+        .grid-item.light path,
+        .grid-item.light rect,
+        .grid-item.light circle,
+        .grid-item.light h2,
+        .grid-item.light p {
```

Comment:
Simplify selector to `.grid-item.light`, and use inheritance for all the rest. IIRC this means adding `.grid-item__link { color: inherit }` and removing `.grid-item__subtitle { color: #FFFFFF }`.